bin/boot

```diff
@@ -54,10 +84,11 @@ then
 else
     file=$ODOO_CONF
 fi
-if [ ! -f "$file" ]; then
-    cp /opt/sources/odoo.conf $file 
-    echo "Copy template as conf file $file"
+if [ -f "$file" ]; then
+    rm $file
 fi
+cp /opt/sources/odoo.conf $file
+echo "Copy template as conf file $file"
```

Comment:
Move this message on top of the command.
Also, please add a log for the chown command (it's usually very long)